amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -314,24 +308,30 @@ export class TableDetail extends React.Component<
                       </time>
                     </section>
                   )}
-                  <section className="metadata-section">
-                    <div className="section-title title-3">Frequent Users</div>
-                    <FrequentUsers readers={data.table_readers} />
-                  </section>
-                  {this.renderProgrammaticDesc(
-                    data.programmatic_descriptions.left
+                  {!data.is_view && (
+                    <section className="metadata-section">
+                      <div className="section-title title-3">Date Range</div>
```

Comment:
Maybe a good chance to move strings to the constants file or updating the text styles?